reference/Chronicler2.v0.yaml

```diff
@@ -5,41 +5,36 @@ info:
   title: Chronicler 2
   version: '0.0'
   summary: An API that shows periodic information regarding Blaseball.
-  description: An API that shows periodic information regarding Blaseball.
+  description: |-
+    An API that shows periodic information regarding Blaseball.
+
+    Currently there are 3 APIs:
+
+    | Name | Description | Kinded |
+    | :-- | :-- | :-- |
+    | [/game-events](Chronicler2.v0.yaml/paths/~1game-events) | Regarding events that happened during games | No |
+    | [/versions](Chronicler2.v0.yaml/paths/~1versions) | Retrieve *chronological* data for a given kind of entity. | Yes |
+    | [/entities](Chronicler2.v0.yaml/paths/~1entities) | Retrieve *latest* data for a given kind of entity. | Yes |
+
+    **Kinded** APIs return different kinds of data based on the value behind `?kind=` - For example, to get players you do `/versions?kind=player`. As a result, they *require* a kind to be passed.
+
+    `/versions` should be used if you want to show how data changed over time - whereas `/entities` should be used when you only care about the *latest* data.
+
+    Each response also has common data like the following:
+
+    | attribute | description |
+    | --------: | :---------- |
+    | `next_page` | used to get the next page using `?page=` |
+    | `items[...].kind` | the kind of the data |
+    | `items[...].entity_id` | ID of the entity - useful to get data for a specific entity - such as a player's data over time. |
+    | `items[...].valid_from` | when the data was taken |
+    | `items[...].valid_to` | when the data became out of date |
   contact:
     name: Astrid
     url: 'https://github.com/xSke/Chronicler/tree/main'
```

Comment:
this is chronicler 1, not 2

Reply:
Could you direct me to the Chronicler 2 repository? Couldn't find it.

Reply:
https://github.com/xSke/chron2

Reply:
Done.